src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1242,10 +1266,28 @@ def test_module_inline_same_kernel():
     # check that the subroutine has only been inlined once
     count = count_lines(psy.gen, "SUBROUTINE time_smooth_code(")
     assert count == 1, "Expecting subroutine to be inlined once"
+    # No compilation test here, see test_module _inline_and_compile
+
+
+@pytest.mark.xfail(reason="Inline function uses a module variable")
```

Comment:
Please could you create an associated Issue for this - e.g. "Validate method of kernel-inline transformation does not check for module variables" - and reference it here.

Reply:
Done, #315

Reply:
Thanks. Please could you put the ref to #315 inside the `reason` string of the xfail.